src/components/index.js

```diff
@@ -1,2 +1,3 @@
 export { default as Footer } from "./Footer";
 export { default as LinksSection } from "./LinksSection";
+export { default as Navbar } from "./Navbar";
```

Comment:
Are we planning to add the main site navbar here as well? if yes, should we rename this navbar to SubsiteNavbar or Navbar2 or something?

Reply:
we won't add the main site Navbar here. this component library will only be used by the country sites. the main site we can keep the components there separate.